.github/workflows/build-wheel-macos-arm64.yaml

```diff
@@ -32,12 +32,6 @@ jobs:
     steps:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v3
-    
-    - name: Setup Runner Environment
```

Comment:
this should not be removed if you have included #409.

Reply:
@maliasadi I think you are commenting on an old commit!